app/src/main/kotlin/com/wire/android/ui/userprofile/UserProfileScreen.kt

```diff
@@ -328,6 +336,46 @@ private fun OtherAccountItem(account: OtherAccount, onClick: (String) -> Unit =
     }
 }
 
+@Composable
+private fun DialogContent(
+    dialogState: DialogState,
+    onDismiss: () -> Unit = {},
+    onStatusChange: (UserStatus) -> Unit = {},
+    onCheckBoxChange: (Boolean) -> Unit = {}
+) {
+    if (dialogState is DialogState.StatusChange) {
+        WireDialog(
+            title = stringResource(id = dialogState.title),
+            text = stringResource(id = dialogState.text),
+            onDismiss = onDismiss,
+            confirmButtonProperties = WireDialogButtonProperties(
+                onClick = { onStatusChange(dialogState.status) },
+                text = stringResource(id = R.string.label_ok),
+                type = WireDialogButtonType.Primary,
+            ),
+            dismissButtonProperties = WireDialogButtonProperties(
+                onClick = onDismiss,
+                text = stringResource(id = android.R.string.cancel),
+                type = WireDialogButtonType.Secondary,
+            )
+        ) {
+            Row {
+                Checkbox(
+                    modifier = Modifier.align(Alignment.CenterVertically),
```

Comment:
I think it's better to set `verticalAlignment = Alignment.CenterVertically` to the `Row` instead of setting it to its every child 😄